includes/Checker/Runtime_Environment_Setup.php

```diff
@@ -100,4 +100,49 @@ public function cleanup() {
 			}
 		}
 	}
+
+	/**
+	 * Checks if the WordPress Environment can be set up for runtime checks.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return bool Returns true if the runtime environment can be set up, false if not.
+	 */
+	public function can_set_up() {
+		global $wp_filesystem;
+
+		require_once ABSPATH . '/wp-admin/includes/upgrade.php';
+
+		if ( ! is_object( $wp_filesystem ) && ! WP_Filesystem() ) {
+			return false;
+		}
+
+		// Check if the object-cache.php file exists.
+		if ( $wp_filesystem->exists( WP_CONTENT_DIR . '/object-cache.php' ) ) {
+			// Check If the object-cache.php file is the Plugin Check version.
+			if ( defined( 'WP_PLUGIN_CHECK_OBJECT_CACHE_DROPIN_VERSION' ) && WP_PLUGIN_CHECK_OBJECT_CACHE_DROPIN_VERSION ) {
+				return true;
+			}
+		} else {
+			// Get the correct Plugin Check directory when run too early.
+			if ( ! defined( 'WP_PLUGIN_CHECK_PLUGIN_DIR_PATH' ) ) {
+				$plugins_dir       = defined( 'WP_PLUGIN_DIR' ) ? WP_PLUGIN_DIR : WP_CONTENT_DIR . '/plugins';
+				$object_cache_copy = $plugins_dir . '/plugin-check/object-cache.copy.php';
```

Comment:
Like in a few other places before, keep in mind that `plugin-check` is not guaranteed to be the folder name. Since we're only dealing with paths here, I'd say a simpler and more reliable solution would be to use `dirname( dirname( __DIR__ ) ) . '/plugin-check/object-cache.copy.php'`.

Reply:
Thanks! Updated in https://github.com/10up/plugin-check/pull/173/commits/8717db9cd7e7c87099df67324a70187716cd52ea